packages/components/textarea/src/textarea.spec.ts

```diff
@@ -0,0 +1,351 @@
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit';
+import '../register.js';
+import { spy, stub } from 'sinon';
+import { ResizeType, Textarea, WrapType } from './textarea';
+
+describe('sl-textarea', () => {
+  let el: Textarea;
+
+  describe('defaults', () => {
+    beforeEach(async () => {
+      el = await fixture(html`<sl-textarea></sl-textarea>`);
+    });
+
+    it('should render correctly', () => {
+      expect(el).shadowDom.to.equalSnapshot();
+    });
+
+    it('should have a textarea slot', () => {
+      const slot = el.renderRoot.querySelector('slot[name="textarea"]');
+
+      expect(slot).to.not.be.null;
+      expect(slot).to.have.attribute('name', 'textarea');
+    });
+
+    it('should not be disabled', () => {
+      expect(el).not.to.have.attribute('disabled');
+    });
+
+    it('should be disabled if set', async () => {
+      el.disabled = true;
+      await el.updateComplete;
+
+      expect(el).to.have.attribute('disabled');
+    });
+
+    it('should not have a value by default', () => {
+      expect(el).not.to.have.attribute('value');
+    });
+
+    it('should have a value when set', async() => {
+      el.value = "my value";
+      await el.updateComplete;
+
+      const textarea = el.querySelector('textarea');
+
+      expect(textarea?.value).to.equal('my value');
+    });
+
+    it('should have a medium size by default', () => {
+      expect(el).to.have.attribute('size', 'md');
+    });
+
+    it('should have a large size when set', () => {
+      el.setAttribute('size', 'lg');
+
+      expect(el).to.have.attribute('size', 'lg');
+    });
+
+    it('should have a wrap property of soft by default', () => {
+      expect(el).to.have.attribute('wrap', 'soft');
+    });
+
+    it('should have a wrap property of hard when set', () => {
+      el.setAttribute('wrap', 'hard');
+
+      expect(el).to.have.attribute('wrap', 'hard');
```

Comment:
does this actually test something? you only check if the attribute is what you just set it, not anything it affects.
If I do 
```
el.setAttribute('blablabla', 'hard');
expect(el).to.have.attribute('blablabla', 'hard');
```
it should pass as well, but i haven't really tested anything